electron/config.ts

```diff
@@ -186,12 +188,41 @@ abstract class GlobalConfig {
       }
     })
 
+    const crossover: Set<WineInstallation> = new Set()
```

Comment:
I think for the lack of organization, we could put this logic into a separate function like we are doing with the`getDefaultWine` so maybe add one called `getCrossover` or something like this. 👍🏽

Reply:
Good point, I'll make that change too.